### ISSUE TITLE
Issues with project

### ISSUE THREAD

**Issue by imakin** [codebrainz/overview-plugin#10]
When i work with projects, closing project doesn't close the opened files, but it records the project as if it has no files instead. So when the project is reopened after restarting geany, no file is present

tested on recent commit 3ca945a geany 1.23.1

**Comment by codebrainz** [comment]
Does it leave any suspicious output in the debug messages (Help->Debug Messages) ? Also, would it be possible to try it with latest Geany 1.24, and/or without any other plugins enabled?

**Comment by codebrainz** [comment]
I can't seem to reproduce.

**Comment by imakin** [comment]
ah yes, i have my own experimental geany but seems i cant install the plugin there even though i have set prefix on configure. So the overview-plugin just installed on my main geany 1.23,
i'll check that later

**Comment by codebrainz** [comment]
@b4n added the `--with-geany-libdir` option to the build system for this. To get a non-standard prefix working here I had to use `PKG_CONFIG_PATH` to make it able to find the correct `geany` pkg-config file, but I think it's supposed to work with only using `--with-geany-libdir`.

**Comment by b4n** [comment]
> @b4n added the `--with-geany-libdir` option to the build system for this. To get a non-standard prefix working here I had to use `PKG_CONFIG_PATH` to make it able to find the correct `geany` pkg-config file, but I think it's supposed to work with only using `--with-geany-libdir`.

Well, not really: the goal of `--with-geany-libdir` is to override installing in the found Geany's libdir.  This is useful for distcheck or packagers, because they don't want to install outside a specific tree, and know what they are doing.

To build for a particular Geany installation, you still need to set `PKG_CONFIG_PATH` as appropriate for it to find the right installation, otherwise even if you install somewhere else, it'd still be built for the found installation.

Bottom line is, do not use `--with-geany-libdir` if you just want to use the built plugin.  If you think you need it for this, you actually need to alter the environment so the build system finds the right Geany installation (using `PKG_CONFIG_PATH`)
